pkg/setup/templates/1.13.go

```diff
@@ -0,0 +1,719 @@
+      containers:
+      - name: kube-proxy
+        image: "{{ .HyperkubeImageURL }}"
+        imagePullPolicy: IfNotPresent
+        command:
+        - /hyperkube
+        - proxy
+        - --config=/var/lib/kubernetes/config.yaml
+        securityContext:
+          privileged: true
+        volumeMounts:
+        - name: config
+          mountPath: /var/lib/kubernetes/
+        livenessProbe:
+          httpGet:
+            path: /healthz
+            port: 10256
+        readinessProbe:
+          httpGet:
+            path: /healthz
+            port: 10256
+        resources:
+          requests:
+            cpu: "50m"
+          limits:
+            cpu: "100m"
+      volumes:
+      - name: config
+        configMap:
+          name: kube-proxy
+`),
+		},
+		{
+			Name:        "p8s-user-admin.yaml",
+			Destination: ManifestAPI,
+			Content: []byte(`---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRoleBinding
+metadata:
+  name: p8s-admin
+roleRef:
+  apiGroup: rbac.authorization.k8s.io
+  kind: ClusterRole
+  name: cluster-admin
+subjects:
+- apiGroup: rbac.authorization.k8s.io
+  kind: User
+  name: p8s
+`),
+		},
+		{
+			Name:        "coredns.yaml",
+			Destination: ManifestAPI,
+			Content: []byte(`---
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: coredns
+  namespace: kube-system
+---
+apiVersion: rbac.authorization.k8s.io/v1beta1
+kind: ClusterRole
+metadata:
+  labels:
+    kubernetes.io/bootstrapping: rbac-defaults
+  name: system:coredns
+rules:
+- apiGroups:
+  - ""
+  resources:
+  - endpoints
+  - services
+  - pods
+  - namespaces
+  verbs:
+  - list
+  - watch
+---
+apiVersion: rbac.authorization.k8s.io/v1beta1
+kind: ClusterRoleBinding
+metadata:
+  annotations:
+    rbac.authorization.kubernetes.io/autoupdate: "true"
+  labels:
+    kubernetes.io/bootstrapping: rbac-defaults
+  name: system:coredns
+roleRef:
+  apiGroup: rbac.authorization.k8s.io
+  kind: ClusterRole
+  name: system:coredns
+subjects:
+- kind: ServiceAccount
+  name: coredns
+  namespace: kube-system
+---
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: coredns
+  namespace: kube-system
+data:
+  Corefile: |
+    .:53 {
+        errors
+        log
+        health
+        kubernetes cluster.local {{ .ServiceClusterIPRange }} {
+          pods insecure
+        }
+        prometheus :9153
+        proxy . /etc/resolv.conf 8.8.8.8 8.8.4.4
+        cache 30
+    }
+---
+apiVersion: extensions/v1beta1
+kind: Deployment
+metadata:
+  name: coredns
+  namespace: kube-system
+  labels:
+    dns: coredns
+    kubernetes.io/name: "CoreDNS"
+spec:
+  replicas: 1
+  strategy:
+    type: RollingUpdate
+    rollingUpdate:
+      maxUnavailable: 0
+  selector:
+    matchLabels:
+      dns: coredns
+  template:
+    metadata:
+      labels:
+        dns: coredns
+    spec:
+      serviceAccountName: coredns
+      tolerations:
+        - key: "CriticalAddonsOnly"
+          operator: "Exists"
+      containers:
```

Comment:
I had restarts on my coredns pod (might be due to my setup), the logs are very limited. I could not find ways to have more verbose logs, do you know of any option ?

Reply:
You can alter the behaviour of coredns with the debug plugin, [see](https://github.com/coredns/coredns/tree/master/plugin/debug).
Let me know if it's related to your setup or with pupernetes itself.

Reply:
Haven't been able to get it to restart again. I will keep this option in mind though.